make_level.py

```diff
@@ -141,38 +116,49 @@ def main():
         "level_name: {}".format(level_name),
         "Confirm[y/n] "
     ]), end='')
-
-    choice = input().lower()
-    if choice == 'n':
-        return
+    
+    choice = ''
+    while choice != 'y':
+        choice = input().lower()
+        if choice == 'n':
+            return
```

Comment:
It might be useful to add `print("Aborting, no changes made.")`

Reply:
done